meta-lmp-base/recipes-security/optee/optee-os-fio_3.12.0.bb

```diff
@@ -1,7 +1,5 @@
 require optee-os-fio.inc
 
 PV = "3.12.0+git"
-SRCREV = "05517fbc0d732ea9a50d14569b1711b058e2d818"
+SRCREV = "4832354e675cdeb01d55ce3a481b1c3d4cf4084d"
 SRCBRANCH = "3.12+fio"
-
-DEFAULT_PREFERENCE = "-1"
```

Comment:
We will require the SE050 integration to be done as part of this PR as well, in order for us to switch to 3.12 by default.